### PR TITLE
Add sorting for report attachments

### PR DESCRIPTION
**Pull request by jimmylam**
Provide sorting for the listing of each type of reports in the program outcome.

### CODE REVIEW

**Comment by halloffame** [comment]
@jimmylam the `sort_by_year_desc` method should probably be a private method at the bottom of the file. While you are at it can you remove the tab from line 17 https://github.com/biola/biola-edu/blob/outcomes-report-order/app/controllers/outcomes_program_controller.rb#L17

Thanks!

**Comment by jimmylam** [comment]
@halloffame  thanks for the comment.  I have made the changes.

File: app/controllers/outcomes_program_controller.rb [hunk 1]
```diff
@@ -1,15 +1,23 @@
...
       @menu = Menu.where(slug: 'outcomes').first
       @departments = Department.published.where(related_object_tags: @program.my_object_tag)
-	 end
+   end
   end
+
+  private
+
+  def sort_by_year_desc(array)
+    arrary.sort {|x,y| y.metadata["year"]<=>x.metadata["year"]}
```
Comment (halloffame) [inline app/controllers/outcomes_program_controller.rb]: @jimmylam Looks like you misspelled `array`.

**Comment by jimmylam** [comment]
@halloffame   oh, man.  Thanks for getting that.

**Comment by halloffame** [comment]
:+1:

**Pull request merged**